### ISSUE TITLE
[Bug][a11y]: Tooltip content not hoverable

### ISSUE THREAD

**Issue by dineshy87** [adobe/spectrum-web-components#3366]
### Code of conduct

- [X] I agree to follow this project's code of conduct.

### Impacted component(s)

Tooltip

### Expected behavior

When a user hovers over an element that triggers a tooltip, the tooltip should appear. The tooltip should remain visible and accessible when the user moves their mouse pointer onto the tooltip itself. This should allow users to interact directly with the content inside the tooltip, such as clicking on a link within the tooltip. The tooltip should only disappear when the user moves their mouse pointer away from both the tooltip and the triggering element, or if the user explicitly dismisses the tooltip (e.g., using the Esc key).

### Actual behavior

When a user navigates to an element with a tooltip and hovers over it, the tooltip appears as expected. However, when the user moves their mouse pointer onto the tooltip itself, the tooltip disappears. This makes it impossible for users to interact directly with the content inside the tooltip.

### How can we reproduce this issue?

1. Navigate to the element triggering the tooltip
2. Hover over the element to trigger the tooltip
3. Try moving the mouse pointer onto the tooltip content itself

Result:
The tooltip disappears as soon as the mouse pointer leaves the boundary of the triggering element and enters the tooltip.

**Comment by Westbrook** [comment]
Interesting concept. We don't count it as appropriate for `sp-tooltip` elements to accept interactive content. We don't block it per say, as it is just an open `<slot>` but we don't support interacting with it by choosing not to allow the pointer for focus to enter the content. This not only reduces the accessibility responsibilities of that content, but simplified touch interactions as well.

Do you see that as not an acceptable path forward? Non-`<sp-tooltip>` content _could_ have interactive content, in which case it should not leverage the default `hover` interactions in our `<overlay-trigger>` element, as it would need a more advanced interaction model. Maybe the right path is to more clearly document that difference?

**Comment by dineshy87** [comment]
By interact i didn't necessarily mean that the tooltip is expected to have interactive content. What i noticed during testing was that i was able to open the tooltip by hovering on the triggering button, but once the tooltip is open, i was not able to move the pointer onto the tooltip. It was closing. The issue here is lack of hoverability of the tooltip. Which was one of the [testing criteria ](https://accessibility.corp.adobe.com/testing/web/manual_checklist/#p_hover)
